INFO 2025-04-01 05:29:50 FaceAlignModelLoader.py: 26] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 05:29:50 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
ERROR 2025-04-01 05:29:50 FaceAlignModelLoader.py: 35] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-04-01 05:30:50 FaceAlignModelLoader.py: 26] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 05:30:50 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
ERROR 2025-04-01 05:30:50 FaceAlignModelLoader.py: 35] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-04-01 05:35:04 FaceAlignModelLoader.py: 27] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 05:35:04 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
ERROR 2025-04-01 05:35:04 FaceAlignModelLoader.py: 36] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-04-01 05:38:17 FaceAlignModelLoader.py: 27] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 05:38:17 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
ERROR 2025-04-01 05:38:17 FaceAlignModelLoader.py: 37] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-04-01 05:43:57 FaceAlignModelLoader.py: 27] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 05:43:57 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 05:43:57 FaceAlignModelLoader.py: 41] Successfully loaded the face landmark model!
INFO 2025-04-01 05:43:57 FaceRecModelLoader.py: 18] Start to analyze the face recognition model, model path: face_sdk/models, model category: face_recognition，model name: face_recognition_1.0
INFO 2025-04-01 05:43:57 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
ERROR 2025-04-01 05:43:57 FaceRecModelLoader.py: 28] The model failed to load, please check the model path: face_sdk/models\face_recognition\face_recognition_1.0\face_recognition_mv.pkl!
INFO 2025-04-01 05:46:55 FaceAlignModelLoader.py: 32] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 05:46:55 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 05:46:55 FaceAlignModelLoader.py: 46] Successfully loaded the face landmark model!
INFO 2025-04-01 05:46:55 FaceRecModelLoader.py: 18] Start to analyze the face recognition model, model path: face_sdk/models, model category: face_recognition，model name: face_recognition_1.0
INFO 2025-04-01 05:46:55 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
ERROR 2025-04-01 05:46:55 FaceRecModelLoader.py: 28] The model failed to load, please check the model path: face_sdk/models\face_recognition\face_recognition_1.0\face_recognition_mv.pkl!
INFO 2025-04-01 05:48:52 FaceAlignModelLoader.py: 32] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 05:48:52 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 05:48:52 FaceAlignModelLoader.py: 46] Successfully loaded the face landmark model!
INFO 2025-04-01 05:48:52 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: face_sdk/models, model category: face_recognition，model name: face_recognition_1.0
INFO 2025-04-01 05:48:52 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 05:48:52 FaceRecModelLoader.py: 38] Successfully loaded the face recognition model!
INFO 2025-04-01 05:53:48 FaceAlignModelLoader.py: 32] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 05:53:48 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 05:53:48 FaceAlignModelLoader.py: 46] Successfully loaded the face landmark model!
INFO 2025-04-01 05:53:48 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: face_sdk/models, model category: face_recognition，model name: face_recognition_1.0
INFO 2025-04-01 05:53:48 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 05:53:48 FaceRecModelLoader.py: 38] Successfully loaded the face recognition model!
INFO 2025-04-01 05:54:47 FaceAlignModelLoader.py: 32] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 05:54:47 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 05:54:47 FaceAlignModelLoader.py: 46] Successfully loaded the face landmark model!
INFO 2025-04-01 05:54:47 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: face_sdk/models, model category: face_recognition，model name: face_recognition_1.0
INFO 2025-04-01 05:54:47 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 05:54:47 FaceRecModelLoader.py: 38] Successfully loaded the face recognition model!
INFO 2025-04-01 05:55:00 FaceAlignModelLoader.py: 32] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 05:55:00 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 05:55:00 FaceAlignModelLoader.py: 46] Successfully loaded the face landmark model!
INFO 2025-04-01 05:55:00 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: face_sdk/models, model category: face_recognition，model name: face_recognition_1.0
INFO 2025-04-01 05:55:00 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 05:55:00 FaceRecModelLoader.py: 38] Successfully loaded the face recognition model!
INFO 2025-04-01 06:03:13 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: models, model category: face_recognition，model name: mobilefacenet
ERROR 2025-04-01 06:03:13 BaseModelLoader.py: 35] The configuration file meta.json was not found or failed to parse the file!
INFO 2025-04-01 06:05:29 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: models, model category: face_recognition_2.0，model name: mobilefacenet
ERROR 2025-04-01 06:05:29 BaseModelLoader.py: 35] The configuration file meta.json was not found or failed to parse the file!
INFO 2025-04-01 06:07:02 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: models, model category: face_recognition，model name: face_recognition_2.0
ERROR 2025-04-01 06:07:02 BaseModelLoader.py: 35] The configuration file meta.json was not found or failed to parse the file!
INFO 2025-04-01 06:08:02 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: models, model category: face_recognition，model name: face_recognition_2.0
ERROR 2025-04-01 06:08:02 BaseModelLoader.py: 35] The configuration file meta.json was not found or failed to parse the file!
INFO 2025-04-01 06:10:26 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: face_sdk\models, model category: face_recognition，model name: face_recognition_2.0
INFO 2025-04-01 06:10:26 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 06:10:26 FaceRecModelLoader.py: 38] Successfully loaded the face recognition model!
INFO 2025-04-01 06:12:43 FaceAlignModelLoader.py: 32] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-04-01 06:12:43 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 06:12:44 FaceAlignModelLoader.py: 46] Successfully loaded the face landmark model!
INFO 2025-04-01 06:12:44 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: face_sdk/models, model category: face_recognition，model name: face_recognition_1.0
INFO 2025-04-01 06:12:44 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 06:12:44 FaceRecModelLoader.py: 38] Successfully loaded the face recognition model!
INFO 2025-04-01 06:15:43 FaceRecModelLoader.py: 23] Start to analyze the face recognition model, model path: face_sdk\models, model category: face_recognition，model name: face_recognition_2.0
INFO 2025-04-01 06:15:43 BaseModelLoader.py: 41] Successfully parsed the model configuration file meta.json!
INFO 2025-04-01 06:15:43 FaceRecModelLoader.py: 38] Successfully loaded the face recognition model!
